Library/Formula/slimerjs.rb

```diff
@@ -11,9 +31,12 @@ class Slimerjs < Formula
     sha1 "aa5654afdd8d2dc049878d2898b9fb7fa33911ab" => :mountain_lion
   end
 
-  if MacOS.version > :snow_leopard
-    option "without-xulrunner", "Build without xulrunner (requires a installed Firefox)"
-    depends_on "xulrunner" => :recommended
+  option "with-xulrunner", "Build with xulrunner"
+
+  if build.with? "xulrunner" && MacOS.version > :snow_leopard
```

Comment:
I think in that case it may be most clear to do something like:

```
depends_on MacOS.version >= :snow_leopard

if build.without? "xulrunner"
  depends_on FirefoxDependency
else
  depends_on "xulrunner"
end
```